src/main/java/cinemas/controllers/user/TheatersController.java

```diff
@@ -0,0 +1,25 @@
+package cinemas.controllers.user;
+
+import cinemas.dtos.CityDto;
+import cinemas.services.CitiesService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+import java.util.List;
+
+@Controller
+@RequestMapping("/theaters")
+public class TheatersController {
+    @Autowired
+    private CitiesService citiesService;
+
+    @GetMapping(value = "")
```

Comment:
```suggestion
    @GetMapping
```
ko có thì vứt luôn dc bé